cmd/status.go

```diff
@@ -37,6 +37,7 @@ func setupStatusCommand() *cobraext.Command {
 		RunE:  statusCommandAction,
 	}
 	cmd.Flags().BoolP(cobraext.ShowAllFlagName, "a", false, cobraext.ShowAllFlagDescription)
+	cmd.Flags().String(cobraext.StackVersionFlagName, "", cobraext.StackVersionFlagDescription)
```

Comment:
This flag might be misleading here. In `elastic-package status apache --version 8.0.0` the version flag may suggest that this is Apache 8.0.0, not Kibana.

What do you think about replacing it with `--kibana-version`?

Reply:
Or `kibana.version`? Thinking of it as the filters you can apply on epr, for example https://epr.elastic.co/search?experimental=true&kibana.version=8.0.0 It would become a more generic feature ...

Reply:
I think that `--kibana-version` will look more like a command option. I haven't seen options with dots, I guess it's a standard? 

But calling for a query is also an option: `--query="kibana.version=8.0.0"`.

Reply:
BTW do we really need the experimental flag? It's always enabled...

Reply:
I used `--version` because it is the flag used to specify the stack version in the stack subcomand, but I see how it can be confusing here.

I have opted for `--kibana-version`, I also prefer it to the version with the dot. How the kibana version is finally encoded in the query is an implementation detail.

> But calling for a query is also an option: `--query="kibana.version=8.0.0"`.

`--kibana-version` covers a very specific use case, so I prefer to have a specific flag for that. Also, package developers don't need to know the registry API to use this command, so I don't think we need generic options by now.

Humans needing to write url query parameters is a call for problems.

> I haven't seen options with dots, I guess it's a standard?

This is not very common, but I am pretty sure it is possible to have options with dots, take a look to `metricbeat --help` :slightly_smiling_face:

> BTW do we really need the experimental flag? It's always enabled...

I hope we can get rid of this soon https://github.com/elastic/package-spec/issues/225 :fire:

Reply:
Thanks for adjusting this!